Formula/clojure-lsp.rb

```diff
@@ -30,14 +30,18 @@ def install
   test do
     require "Open3"
 
-    stdin, stdout, _, wait_thr = Open3.popen3("#{bin}/clojure-lsp")
+    system "file", "#{bin}/clojure-lsp"
+    stdin, stdout, _stderr, wait_thr = Open3.popen3("#{bin}/clojure-lsp 2>&1")
```

Comment:
```suggestion
    stdin, stdout, _, wait_thr = Open3.popen3("#{bin}/clojure-lsp 2>&1")
```

You don't use `stderr`, so there's no point assigning to it. Please squash your commits once you've done this. Thanks!